internal/snmp/translate.go

```diff
@@ -40,14 +69,14 @@ func LoadMibsFromPath(paths []string, log telegraf.Logger) error {
 		if err != nil {
 			return fmt.Errorf("Filepath could not be walked %v", err)
 		}
+
 		for _, folder := range folders {
 			err := filepath.Walk(folder, func(path string, info os.FileInfo, err error) error {
 				// checks if file or directory
 				if info.IsDir() {
-					gosmi.AppendPath(path)
+					appendPath(path)
 				} else if info.Mode()&os.ModeSymlink == 0 {
-					_, err := gosmi.LoadModule(info.Name())
-					if err != nil {
+					if err := loadModule(info.Name()); err != nil {
 						log.Warnf("Module could not be loaded %v", err)
```

Comment:
```suggestion
						log.Warn(err)
```
The error from gosmi already has the same/similar text.